src/vs/editor/contrib/inlineCompletions/browser/ghostTextWidget.ts

```diff
@@ -274,6 +275,10 @@ class DecorationsWidget implements IDisposable {
 		// Using change decorations ensures that we update the id's before some event handler is called.
 		this.editor.changeDecorations(accessor => {
 			this.decorationIds = accessor.deltaDecorations(this.decorationIds, parts.map<IModelDeltaDecoration>(p => {
+				if (this.editor.getOption(EditorOption.ariaAssertiveInlineSuggestion)) {
+					this._alertWithContent(withNullAsUndefined(document.querySelector('.ghost-text-decoration')?.textContent));
```

Comment:
Is this the best way to get the current inline completions? Can we somehow get it from the model, and not from HTML?

Reply:
Notice that the suggested text might have holes, e.g. in an if statement:
```
if (|[x === 1])[ { }]
```

The cursor is at `|`, everything in `[`...`]` is a suggestion.

You should probably use `parts` to figure out what to read.